pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2024-04-17T15:45:00.000Z'
+category: announcements
+title: 'Node.js 22 is now available!'
+layout: blog-post
+author: The Node.js Project
+---
+
+We're excited to announce the release of Node.js 22!
+Highlights include require()ing ESM graphs, WebSocket client, updates of the V8 JavaScript engine, and more!
+
+As a reminder, Node.js 22 will enter long-term support (LTS) in October, but until then, it will be the "Current" release for the next six months.
+We encourage you to explore the new features and benefits offered by this latest release and evaluate their potential impact on your applications.
+
+The project continues to make progress across a number of areas, with many new features and fixes flowing into existing LTS releases.
+For that reason, the changes outlined in the [changelog][CHANGELOG] for Node.js 22 only represent a small subset of the features and
+work since the last major release. This blog post will add some additional context on the larger body of work in relation to those changes.
+
+You can read more about our release policy at <https://github.com/nodejs/release>.
+
+To download Node.js 22.0.0, visit: [https://nodejs.org/download/current/](/download/current/). You can find the release post at [https://nodejs.org/blog/release/v22.0.0](/blog/release/v22.0.0),
+which contains the full list of commits included in this release.
+
+### Notable Changes
+
+#### glob and globSync
```

Comment:
This section should mention that these belong to the `node:fs` module IMO. (They don't do arbitrary pattern-matching.)